tests/plugins/autogrow/autogrowwidth.js

```diff
@@ -0,0 +1,37 @@
+/* bender-tags: editor, autogrow */
+/* bender-ckeditor-plugins: autogrow */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
```

Comment:
It might be goo to set editor height and width explicitly so it is not influenced by any other plugins (e.g. in full presets tests).

Reply:
It seems that setting width explicitly does not break editor width. I'm for not setting editor size here. WDYT?

Reply:
Ok, so setting editor width explicitly is also a workaround for this issue :+1:

Let's leave it without explicit width so we are sure test checks this regression.